code/src/App.js

```diff
@@ -1,9 +1,28 @@
-import React from 'react'
+import React, { useState } from 'react'
+import musicData from './data.json'
+import { Header } from './components/Header'
+import { Album } from './components/Album'
+import { EmptyState } from './components/EmptyState'
 
 export const App = () => {
+  const [data, setData] = useState(musicData.albums.items);
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <Header toChild={data} sendToParent={setData} />
```

Comment:
The way you name your props could be improved. We don't have to specify that one will be send "toChild" and another one "toParent", because it always works like this. Instead, we should rather aim for names as descriptive as we can. What could be good example in your case? I would suggest something like this :

```
<Header albums={data} updateAlbums={setData} />
```